tailwind.config.js

```diff
@@ -16,21 +16,23 @@ module.exports = {
             md: { min: '800px' }
         },
         colors: {
+            ////from tailwind labs https://youtu.be/MAtaT8BZEAo. allows opacity to still work with css vars.
+            //https://tailwindcss.com/docs/customizing-colors#using-css-variables
             transparent: 'transparent',
             current: 'currentColor',
             currentColor: '#ED6C5A',
-            orange: {
-                100: '#fff7ed',
-                200: '#ffedd5',
-                300: '#fed7aa',
-                400: '#fb923c',
-                DEFAULT: '#ED6C5A',
-                500: '#ED6C5A',
-                600: '#ea580c',
-                700: '#c2410c',
-                800: '#9a3412',
-                900: '#7c2d12'
-            },
+            primary: 'rgb(var(--color-primary) / <alpha-value>)',
```

Comment:
added support for multiple themes! 
I know we don't need this right now, but I had some fun learning about it.